src/workers/eth/EthUpdater.ts

```diff
@@ -388,6 +440,17 @@ export class EthUpdater {
             await this.processSync(event, domainRepository);
             break;
           }
+          case 'SetReverse': {
+            await this.processSetReverse(event, domainRepository);
+            break;
+          }
+          case 'RemoveReverse': {
+            await this.processRemoveReverse(
+              event,
+              manager.getRepository(DomainsReverseResolution),
```

Comment:
nit: why not having a variable just like `domainRepository` to be consistent?

```suggestion
              const domainReverseResolutionRepository = manager.getRepository(Domain);
              
              this.processRemoveReverse(
                event,
                domainReverseResolutionRepository,
              )
```

Reply:
I don't have any preference, but my reasoning to have it inline was that it's more compact this way.
It doesn't make sense to define it at the beginning of the function like `domainRepository` since it's only used here. 
And if someone ever needs to reuse this repository, they can add a variable where necessary.

If you think it's better to have a variable declared, I'll update it.

Reply:
Make sense. No, I don't have strong preference here either. Let's leave it as it.